src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  if (!robot) {
```

Comment:
If you want to check if the robot is an object there is a better way to do that
```
typeof robot !== 'object' || robot === null
```
Becuz if the robot is a string or a number value you still has false in your condition
[link](https://ru.hexlet.io/qna/javascript/questions/kak-proverit-yavlyaetsya-li-peremennaya-ob-ektom-js)

Reply:
It's just an argument pass check=/

Reply:
I didn't understand what did we check here ?